src/specs/data_structures.md

```diff
@@ -464,26 +463,16 @@ Finally, the `availableDataRoot` of the block [Header](#header) is computed as t
 
 ### Invalid Erasure Coding
 
-If a malicious block producer incorrectly computes the 2D Reed-Solomon code for a block's data, a fraud proof for this can be presented.
+If a malicious block producer incorrectly computes the 2D Reed-Solomon code for a block's data, a fraud proof for this can be presented. Data type for the fraud proof can be found in [here](./networking.md#badencodingfraudproof).
 
 #### ShareProof
 
-| name       | type                                                  | description                                                                                       |
-|------------|-------------------------------------------------------|---------------------------------------------------------------------------------------------------|
-| `share`    | [Share](#share)                                       | The share.                                                                                        |
-| `proof`    | [NamespaceMerkleTreeProof](#namespacemerkletreeproof) | The Merkle proof of the share in [`availableDataRoot`](#header).                                  |
-| `isCol`    | `bool`                                                | A Boolean indicating if the proof is from a row root or column root; `false` if it is a row root. |
-| `position` | `uint64`                                              | The index of the share in the offending row or column.                                            |
-
-#### BadEncodingFraudProof
-
-| name          | type                                                  | description                                                                       |
-|---------------|-------------------------------------------------------|-----------------------------------------------------------------------------------|
-| `shareProofs` | [ShareProof](#shareproof)`[]`                         | The available shares in the offending row or column.                              |
-| `root`        | [HashDigest](#hashdigest)                             | The Merkle root of the offending row or column.                                   |
-| `proof`       | [NamespaceMerkleTreeProof](#namespacemerkletreeproof) | The Merkle proof of the row or column root in [`availableDataRoot`](#header).     |
-| `isCol`       | `bool`                                                | A Boolean indicating if it is an offending row or column; `false` if it is a row. |
-| `position`    | `uint64`                                              | The index of the row or column in the square.                                     |
+| name       | type                                                           | description                                                                                       |
```

Comment:
Running the formatting on all these tables show they're not formatted with [Markdown table prettifier](https://github.com/darkriszty/MarkdownTablePrettify-VSCodeExt).

Reply:
Fixed.